src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -234,10 +233,8 @@ export const PublishingAccordion = ({
           <LoadingButton
             disabled={isLoading !== LoadingState.None || !registrationIsValid}
             data-testid={dataTestId.registrationLandingPage.tasksPanel.publishButton}
-            sx={{ mt: '1rem' }}
-            color="primary"
-            variant="contained"
-            endIcon={<CloudUploadIcon />}
+            sx={{ mt: '1rem', bgcolor: 'white' }}
```

Comment:
pedantisk: men er ikke det som er trendy blant kidsa er å lese ut bakgrunnsfargen til temaet til brukeren.
Bare lurer.

Reply:
Kidsa elsker dynamisk background (er ihvertfall det som er word on the street), men blir litt clash med resten av designet🤔